storage/mock.go

```diff
@@ -19,24 +20,40 @@ const (
 	MockMBObject = "mb"
 	MockGBObject = "gb"
 	MockTBObject = "tb"
+
+	MockPek3a = "mock-pek3a"
```

Comment:
We do not use valid zone name in mock, instead, we can use `MockZoneAlpha`, `MockZoneBeta`.

And two zone is enough.